README.md

```diff
@@ -208,17 +171,12 @@ Example:
 
 > Mention all of the collaborators of this project.
 
```

Comment:
## 1.
![image](https://user-images.githubusercontent.com/93039359/224507914-bdca31db-da06-4596-bd40-88dabb6b5c3f.png)
## 2.
![image](https://user-images.githubusercontent.com/93039359/224507944-1c96bac3-378d-4985-be28-5c523210b100.png)
## 3.
![image](https://user-images.githubusercontent.com/93039359/224507959-f1b1d7bc-fd7c-4188-af0c-96da592e0452.png)
## 4.
![image](https://user-images.githubusercontent.com/93039359/224508045-a4277d46-e06d-4387-ac23-876b5db4a88c.png)
## 5.
![image](https://user-images.githubusercontent.com/93039359/224508058-bbe1dfca-e095-4bab-a90e-95aaca471ad3.png)
## 6.
![image](https://user-images.githubusercontent.com/93039359/224508069-3fe76314-b1e7-4878-92e1-d2d7ddac6309.png)

- [ ] It's great that you have deleted some placeholder texts but as suggested by the previous reviewers that you still have some placeholder texts in your readme file which you didn't delete yet 😨 Kindly delete all of them shown in the image above ⬆️ and I would apologize on the behalf of the previous reviewers because they didn't give you proper information 🤔 If you have any question then you can reach out to me on slack by searching my name Hammas in the full-time students channel 👍 or you can call me on zoom 😊